test_scripts/API/VehicleData/StabilityControlsStatus/010_GetVehicleData_StabilityControlsStatus_InvalidParameters.lua

```diff
@@ -0,0 +1,74 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data
+-- (with invalid parameters) via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+--
+-- Steps:
+-- 1) App send GetVehicleData (with stabilityControlsStatus = true) request to SDL
+-- SDL does:
+--  - send VehicleInfo.GetVehicleData (with stabilityControlsStatus = true) request to HMI,
+-- HMI sends VehicleInfo.GetVehicleData response
+--  - with "SUCCESS", stabilityControlsStatus has invalid parameters
+-- SDL does:
+--  - send to app GetVehicleData response with (success: false, resultCode: "GENERIC_ERROR")
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Local Variables ]]
+local reqParams = { 
+  stabilityControlsStatus = true
+}
+
+local hmiResParams_invalidTrailerSwayControl = {
+  stabilityControlsStatus = {
+    escSystem = "ON",
+    trailerSwayControl = true
```

Comment:
```suggestion
 -- value type is updated to boolean instead of VehicleDataStatus type
    trailerSwayControl = true
```
I would propose to add the comment what exactly is invalid, like
-- value type is updated to boolean instead of VehicleDataStatus type

Reply:
@VjKlepikov done in this [commit](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/14/commits/1083dd2dbfb083e5b89ac7e7cf8bd751979f060a)